instructions/docs/encryption-context.md

```diff
@@ -544,6 +600,21 @@ There's a few simple suggestions to get you started in the snippets below.
     # Ctrl-D when finished to exit the REPL
     ```
 
+=== "C#"
+
+    ```bash
+    // Run your code
+    dotnet run
+    
+    // Follow the menu prompts to interact with the document bucket
+    // You can close the program at any time with Ctrl+c
+    
+    // Alternatively, you can edit the Main method in App.cs
+    // to interact with the Api class directly.
+
+    // See if you can modify the code to make the encryption context validation fail!
```

Comment:
They should also see this fail if the EC they supply on retrieve doesn't match the one used with store. I think we should be suggesting people see that in play vs messing with the code they just added.